### PR TITLE
Overrride gte relation for new pipeline in sigclient

### PR DESCRIPTION
**Pull request by vasukalariya**
# Description
Summarize the change.
- Override relation to pass `gte` if its received.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: tools/sigclient/pkg/query/query.go [hunk 1]
```diff
@@ -921,6 +921,9 @@ func RunQueryFromFileAndOutputResponseTimes(dest string, filepath string, queryR
 }
 
 var skipIndexes = map[int]bool{
+	// Working now
+	4: true, // Not working Pipe QL
```
Comment (AndrewHess) [inline tools/sigclient/pkg/query/query.go:925]: Why do you say `Working now` but list it in the indexes to skip?

File: tools/sigclient/pkg/query/query.go [hunk 2]
```diff
@@ -734,17 +734,17 @@ func RunQueryFromFile(dest string, numIterations int, prefix string, continuous,
...
-									var ok bool
-									finalHits, ok = v.(float64)
-									if !ok {
-										log.Fatalf("RunQueryFromFile: Returned total matched is not a float: %v", v)
-									}
-									hits, err = utils.VerifyInequality(finalHits, relation, expectedValue)
-
-								}
+							value := eValue["value"]
+							receivedRelation := eValue["relation"]
```
Comment (AndrewHess) [inline tools/sigclient/pkg/query/query.go:738]:
```suggestion
							value, ok := eValue["value"]
							receivedRelation, ok := eValue["relation"]
```
And check the `ok` values

File: tools/sigclient/pkg/query/query.go (hunk 1 above)
Comment (vasukalariya) [reply tools/sigclient/pkg/query/query.go]: addressed in the latest commit. It is not working

File: tools/sigclient/pkg/query/query.go (hunk 2 above)
Comment (vasukalariya) [reply tools/sigclient/pkg/query/query.go]: addressed in the latest commit.

File: tools/sigclient/pkg/query/query.go [hunk 3]
```diff
@@ -734,17 +734,22 @@ func RunQueryFromFile(dest string, numIterations int, prefix string, continuous,
...
+							if !ok {
+								log.Fatalf("RunQueryFromFile: relation not present in totalMatched")
+							}
+							finalHits, ok = value.(float64)
+							if !ok {
+								log.Fatalf("RunQueryFromFile: Returned total matched is not a float: %v", value)
+							}
+							if receivedRelation == "gte" && relation != "lt" {
+								relation = "gte"
+							}
```
[the remark this answers is not in the document]
Comment (vasukalariya) [reply tools/sigclient/pkg/query/query.go:751]: Actually we want to change this to use explicit check for exact totalMatched and relation. For migration purpose I think we can just override the relation with received relation for `gte`.

**Pull request closed without merging**